### PR TITLE
Removed the space between '@' and the 'userhandle'

### PR DESCRIPTION
**Pull request by pranavgoel29**
## Description
There was some spacing between the @ and the user handle which was not looking good.

### Before
image

### After
image

> Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

### CODE REVIEW

**Comment by proful** [comment]
@PuruVJ can you explain @pranavgoel29 how he can avoid the below stuff

Also read https://github.com/twindle-co/twindle/issues/698

image

File: twindle-cli/renderer/pdf/templates/Thread.hbs [hunk 1]
```diff
@@ -215,8 +216,7 @@
         </svg>
         {{/if}}
         <span class="user-handle">
-          @
-          {{common.user.username}}
+          @{{common.user.username}}
```
Comment (PuruVJ) [inline twindle-cli/renderer/pdf/templates/Thread.hbs:219]: The problem here is that when you format your document, its going to put them on separate lines again, and we'll have the same problems again.

Using handlebars helpers can help with that . See https://stackoverflow.com/questions/29666047/sane-way-to-concat-string-and-variable-in-handlebars-js-helper-argument

File: twindle-cli/twitter/utils/process-tweet-entities.js [hunk 2]
```diff
@@ -182,6 +182,7 @@ function renderMentionsHashtags({ text = "", mentions = [], hashtags = [] }) {
 /**
  * Fix user description from multiple tweets combined obj. DO NOT COMPOSE IN THE RENDERRICHTWEETS FUNCTION
  */
+
```
Comment (PuruVJ) [inline twindle-cli/twitter/utils/process-tweet-entities.js:185]: JSDoc won't work if there's a space in between. PLease remove the blank line

**Review by PuruVJ — CHANGES_REQUESTED** [review]

**Comment by PuruVJ** [comment]
> @PuruVJ can you explain @pranavgoel29 how he can avoid the below stuff
> 
> Also read #698
> 
> image

On it

File: twindle-cli/twitter/utils/process-tweet-entities.js (hunk 2 above)
Comment (pranavgoel29) [reply twindle-cli/twitter/utils/process-tweet-entities.js:185]: Then why there is other blank lines in file 😅

File: twindle-cli/renderer/pdf/templates/Thread.hbs (hunk 1 above)
Comment (pranavgoel29) [reply twindle-cli/renderer/pdf/templates/Thread.hbs:219]: Can you explain a bit more what you are trying to say ?

Comment (PuruVJ) [reply twindle-cli/renderer/pdf/templates/Thread.hbs:219]: What I'm saying is that when you're using something like VSCode with Prettier as a formatter, and you hit your shortcut keys to format your code, prettier beautifies the code, and it sometimes break lines off. That is what is gonna happen. It's going to  put the {{common.user.username}} on a separate line again, and we'll have the space again between the @ and username.

That's why we need to concatenate these 2, like '@' + common.user.username. The SO link I sent you has the code to deal with the concatenation

File: twindle-cli/twitter/utils/process-tweet-entities.js (hunk 2 above)
Comment (PuruVJ) [reply twindle-cli/twitter/utils/process-tweet-entities.js:185]: Please remove this line 185 here.

**Comment by johnjacobkenny** [comment]
Nice spot @pranavgoel29 :+1:

**Pull request closed without merging**